massgov/mayflower
# DP-11301: feedback integration updates

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

This provides feedback form updates to be pulled into the feedback form changes in Drupal.

## Related Issue / Ticket

- [DP-11301](https://jira.mass.gov/browse/DP-11301)

## Steps to Test


- Checkout the mayflower branch
- Build locally
- Confirm that the patterns in Organisms->feedback match intended designs
- Testing in Drupal is also intended for approving this ticket.  See https://jira.mass.gov/browse/DP-11148 for more information on the Drupal testing.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
